src/modules/battery.cpp

```diff
@@ -481,8 +495,15 @@ const std::tuple<uint8_t, float, std::string, float> waybar::modules::Battery::g
     }
 
     float time_remaining{0.0f};
-    if (status == "Discharging" && total_power_exists && total_energy_exists) {
+    if (status == "Discharging" && time_to_empty_now_exists) {
+      if (time_to_empty_now != 0) time_remaining = (float)time_to_empty_now / 1000.0f;
```

Comment:
What is divisor `1000` based on exactly? According to: https://docs.kernel.org/power/power_supply_class.html both `time_to_empty` as well as `time_to_full` are values in seconds and I think `time_remaining` is expected to be in hours so divisor should be `3600` no?

Reply:
@jpalus I'm agree something wrong.

upower: time to empty:       3.8 hours
waybar: 12h14